planet/index.html

```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
 <html>
     <head>
-            <title>Music Blocks is a collection of tools for exploring musical concepts.</title>
+            <title>Music
```

Comment:
Is there a need for a newline here?

Reply:
Sure i will keep it one line online

Reply:
@walterbender  now i have kept it in single line